readers/elasticsearch/src/main/java/com/criteo/hadoop/garmadon/elasticsearch/ElasticSearchReader.java

```diff
@@ -176,10 +210,15 @@ private static BulkProcessor setUpBulkProcessor(String esHost, int esPort, Strin
         //setup es client
         RestHighLevelClient esClient = new RestHighLevelClient(restClientBuilder);
 
+        putGarmadonTemplate(esClient, elasticsearch);
+
         Sniffer sniffer = Sniffer.builder(esClient.getLowLevelClient()).build();
         sniffOnFailureListener.setSniffer(sniffer);
 
-        return BulkProcessor.builder(esClient::bulkAsync, new ElasticSearchListener())
+        BiConsumer<BulkRequest, ActionListener<BulkResponse>> bulkConsumer =
```

Comment:
what is this change for ?

Reply:
Due to bump of ES to 6.5

Reply:
Was just curious since we use RequestOptions.DEFAULT. I'm ok with the way it is done here :)